theano/sandbox/cuda/basic_ops.py

```diff
@@ -3300,23 +3282,20 @@ def make_node(self, value, *shape):
                 const_shp = None
             bcast.append(numpy.all(1 == const_shp))
         otype = CudaNdarrayType(dtype='float32', broadcastable=bcast)
-        return Apply(self, [v] + sh, [otype()])
+        return Apply(self, sh, [otype()])
```

Comment:
To avoid problems with DebugMode, create the output variable (`otype()`) outside of the Apply() call and set its `values_eq_approx` property to a function that takes two inputs and always returns `True`.  This function has to be defined at the file level to be pickleable.